scripts/tests/systemd/firezone-client.service

```diff
@@ -8,7 +8,7 @@ Environment="FIREZONE_API_URL=ws://localhost:8081"
 Environment="FIREZONE_DNS_CONTROL=systemd-resolved"
 Environment="FIREZONE_ID=D0455FDE-8F65-4960-A778-B934E4E85A5F"
 Environment="FIREZONE_TOKEN=n.SFMyNTY.g2gDaANtAAAAJGM4OWJjYzhjLTkzOTItNGRhZS1hNDBkLTg4OGFlZjZkMjhlMG0AAAAkN2RhN2QxY2QtMTExYy00NGE3LWI1YWMtNDAyN2I5ZDIzMGU1bQAAACtBaUl5XzZwQmstV0xlUkFQenprQ0ZYTnFJWktXQnMyRGR3XzJ2Z0lRdkZnbgYAGUmu74wBYgABUYA.UN3vSLLcAMkHeEh5VHumPOutkuue8JA6wlxM9JxJEPE"
-Environment="RUST_LOG=firezone_linux_client=trace,connlib_client_shared=trace,firezone_tunnel=trace,connlib_shared=trace,boringtun=debug,snownet=debug,str0m=info,warn"
+Environment="RUST_LOG=firezone_linux_client=info,connlib_client_shared=info,firezone_tunnel=info,connlib_shared=info,snownet=info,str0m=info,warn"
```

Comment:
Is there anything that gets logged on `INFO` that we don't want to see? Would it make more sense to just default to `INFO` instead and selectively turn off too-noisy logs?